grammar.js

```diff
@@ -61,9 +61,16 @@ module.exports = grammar({
 
     string: ($) =>
       repeat1(
-        choice($._quoted_string, $._unquoted_string, $._line_continuation)
+        choice(
+          $._quoted_string,
+          $._unquoted_string,
+          $._empty_string,
+          $._line_continuation
+        )
       ),
 
+    _empty_string: ($) => prec.left('""'),
```

Comment:
Instead of introducing another node for this I think we can change the `_quoted_string` rule to use `repeat` instead of `repeat1`. That should make the contents of the string optional

Reply:
Indeed, I got confused yesterday by error messages from updating tree-sitter-cli locally. Thanks!